server/plugins/views.js

```diff
@@ -34,8 +35,8 @@ module.exports = {
     context: {
       appVersion: pkg.version,
       assetPath: '/assets',
-      serviceName: config.serviceName,
-      pageTitle: `${config.serviceName}`,
+      serviceName: constants.serviceName,
+      pageTitle: `${constants.serviceName}`,
```

Comment:
Could this be the same as the line above, without the string template?

Reply:
Yup! 🤦

Reply:
I've destructured the objects for consistency with the rest of the code base.